test/index.js

```diff
@@ -1,245 +0,0 @@
-require('should');
-var Limiter = require('..');
```

Comment:
Yep. Sounds good

On Thu, Mar 17, 2016, 13:50 Gildas Garcia <notifications@github.com> wrote:

> In test/index.js
> <https://github.com/tj/node-ratelimiter/pull/21#discussion_r56491715>:
>
> > @@ -1,245 +0,0 @@
> > -require('should');
> > -var Limiter = require('..');
>
> I like it. Maybe status would be better though.
>
> —
> You are receiving this because you commented.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/tj/node-ratelimiter/pull/21/files/68a51eae954c0f9e88b86752badaee0507e9eb24#r56491715>
>